RHEL6_7/packages/RemovedPackages/noauto_postupgrade.d/native_rpm_dep_fix.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+#RHSIGNED_PKGS="/var/cache/preupgrade/common/rpm_rhsigned.log"
+RHSIGNED_PKGS="rpm_rhsigned.log"
+kept_broken_rpms="kept_broken_rpms"
+VERBOSE=0
+RM_CUSTOM_RPM=0
+FAIL=0
+
+log_error()   { echo >&2 "Error:   $@"; }
+log_warning() { echo >&2 "Warning: $@"; }
+log_info()    { echo >&2 "Info:    $@"; }
```

Comment:
Mixing array and string.  Did you mean this?

    log_error() { echo >&2 "Error: " "$@"; }

or (argument is line):

    log_error() { for msg in "$@"; do echo >&2 "Error: $msg"; done; }

or just ignore from $2 on::

    log_error() { echo >&2 "Error: $2"; }

Reply:
The forth option :D :
```
log_error() { echo >&2 "Error: $1"; }
```